compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeBuilder.java

```diff
@@ -5427,18 +5427,22 @@ private boolean isDeclaredWithVar(Node typeNode) {
     }
 
     private BLangIdentifier createIdentifier(Token token) {
-        return createIdentifier(getPosition(token), token);
+        return createIdentifier(getPosition(token), token, false);
     }
 
-    private BLangIdentifier createIdentifier(Location pos, Token token) {
+    private BLangIdentifier createIdentifier(Token token, boolean isXML) {
```

Comment:
Since the `isXML` value is `false` in most cases, won't it be better if we have another overloaded method that takes just the `Location` and `Token` and use that in all other places?

Reply:
Logic change in the method `createIdentifier(Location pos, Token token, boolean isXML)` is minimal, So, is it kind a duplicating the most of the same logic into a separate overloaded method?

Reply:
Updated bf0a9741b71414b86cb2390b41aa910659db6d44